.travis.yml

```diff
@@ -11,4 +11,30 @@ sudo: required
 addons:
   chrome: stable
 
-script: "cd $TEST_DIR && npm install && npm test"
+script: "cd ${TRAVIS_BUILD_DIR}/${TEST_DIR} && npm install && npm test"
+
+env:
+  global:
+    # an encrypted 'GITHUB_TOKEN=<token>' for the PolymerLabs/arcs repo
+    secure: "qFfvf0YAK5NWkH5fj708Z+1qIHtOFrmNhslB+n14rt5J3fJjY0A/4lbLbhx2jACYME+OU2m66maxh5abEZCF4ZGi4zpJa01IUYesTTa3qdM9kozB2TfFcl0d1xffuGpAh9kPchaadCvx/BU9fYFzkd8CLD5mQWCybwqM+6xpoO/KULDXhlcOPGbzg65xmPJANZH+ldH0NgfVLqofeFUg1j0bke1n+7DLf+Oh4ZiYZavciJjhuYx561TkkSEd1gqQGImEfkmKmhz82E8curx6uJW+kD9WagNbLj4FvEsBETIM7BSE8sei2vOVO42OnzvCej+P98IUelEqBS6sTeUNp79f8cC8n7nSYUJwTpshturBFoIFfcVQNoebdaao6gWpnxhUEnsc36AMl0DpIT0DsA9SxZQRjqhAEZmAWmASJMc5t/jWIjmJrqQxDy80HGJPEhIMTOAZfAiFqwyQwYTvJ0tOTW+jk2P9oriLHwi0Pr88uPGD5HWS6ULPoAutQAWMKCYgb5PoOiB8vbUZT9RTXfdLkHLkpw0ICaj5sam2o32CMLm8KZzfNVLJ/kUgdnkYUpC0vo+PGfguiJX8fvZxB06+niljW9iHteqVuoP1tADQwby4jS2ob3eL7TgCGuGk6gQkvHkLmOzlagKeXu1YZ0ow5vq6C+vSEfKEzIDrhXk="
+
+before_deploy:
+  # Travis pages deploys skips artifacts from .gitignore. We need some of those, so
+  # remove them from .gitignore.
+  - cd ${TRAVIS_BUILD_DIR} && sed -i.sedbackup -e 's/^shell\/build$//g' .gitignore
+
+deploy:
+  provider: pages
+  skip-cleanup: true
+  github-token: $GITHUB_TOKEN
+  keep-history: true
```

Comment:
What does the commit/description look like? Ideally it would have two parents, the previous master of arcs-live, and the commit from which the build is being deployed. But as long as we have some connection to the commit from arcs, even if it's just in the description that should be sufficient.

Reply:
It's pretty bare-bones. Check out my test: https://github.com/smalls/arcs-live-test/commit/978a06dde0a07bb3e96bba4fed4051bbf53bbbd6

It has the parent within the deploy repo, but not the reference back to the main coding repo. In order to support releases I'll need to script something more complex and that'll give us additional controls.